cmd/options/options.go

```diff
@@ -24,20 +24,17 @@ import (
 	"net/url"
 
 	"github.com/spf13/pflag"
+
+	"k8s.io/node-problem-detector/pkg/custompluginmonitor"
```

Comment:
This makes it impossible to compile out custompluginmonitor and systemlogmonitor.

I think to handle the deprecated flag, it is fine to hard code the plugin name in this file to get rid of this unnecessary dependency.

Reply:
After this PR is merged, I plan to change the Prow tests that's still using old NPD flags.

Once the testing changes are done, I plan to remove the deprecated flags and all the logic around it (including here). And then we will have the ability to compile out custompluginmonitor and systemlogmonitor.

By "remove", I mean that when you do `./node_problem_detector --help`, those flags won't show up at all. It's like they never existed.

Reply:
You can use `MarkDeprecated` to achieve that, and we should keep those flags for backward compatibility. See https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli

Reply:
Sounds good. Thanks!

Reply:
We should really remove this. :p 

It is unnecessary to break the compile-in plugin system just because of a constant.

I'll send a PR to address this. :)